src/main/resources/openapi.json

```diff
@@ -24,7 +24,7 @@
                 "tags": [
                     "Consumers"
                 ],
-                "description": "Creates a consumer instance in the given consumer group. You can optionally specify a consumer name and supported configuration options. It resturns a base URI used to construct URIs for subsequent requests against this consumer instance.",
+                "description": "Creates a consumer instance in the given consumer group. You can optionally specify a consumer name and supported configuration options. It returns a base URI used to construct URIs for subsequent requests against this consumer instance.",
```

Comment:
```suggestion
                "description": "Creates a consumer instance in the given consumer group. You can optionally specify a consumer name and supported configuration options. It returns a base URI which should be used to construct URIs for subsequent requests against this consumer instance.",
```

Reply:
Should be or must be?

Reply:
"must", since (depending on how it's deployed) it might not work otherwise.